src/Hamiltonians/geometry.jl

```diff
@@ -222,21 +276,118 @@ Base.size(off::Displacements) = (length(off.geometry),)
     end
 end
 
-"""
-    neighbor_site(geom::CubicGrid, site, i)
-
-Find the `i`-th neighbor of `site` in the geometry. If the move is illegal, return 0.
-
-See also [`CubicGrid`](@ref).
-"""
 function neighbor_site(g::CubicGrid{D}, mode, chosen) where {D}
     # TODO: reintroduce LadderBoundaries small dimensions
     return g[g[mode] + Directions(D)[chosen]]
 end
 
+num_neighbors(::CubicGrid{D}) where {D} = 2D
+
 function BitStringAddresses.onr(address, geom::CubicGrid{<:Any,S}) where {S}
     return SArray{Tuple{S...}}(onr(address))
 end
 function BitStringAddresses.onr(address::CompositeFS, geom::CubicGrid)
     return map(fs -> onr(fs, geom), address.components)
 end
+
+"""
+    HoneycombLattice((height, width), fold=(true, true))
```

Comment:
```suggestion
    HoneycombLattice((height, width), fold=(true, true)) <: Geometry{2}
```
A link back to `Geometry` would be helpful.